packages/suite/src/views/wallet/send/components/Outputs/components/Amount/index.tsx

```diff
@@ -219,11 +207,6 @@ export const Amount = ({ output, outputId }: AmountProps) => {
             <Row>
                 <Heading>
                     <Translation id="AMOUNT" />
-                    {tokenBalance && (
```

Comment:
Ok, makes sense. Why I originally deleted it was because on smaller screens with hover it looked bad since switch for send max is there. Texted @Hermez-cz and will improve it with https://github.com/trezor/trezor-suite/issues/10435

Reply:
This mobile screenshot should be fixed from empower guys. oka